File: 백준/Bronze/4344.py
T = int(input())

for tc in range(T):
    scores = list(map(int, input().split()))
    num = 0
    avg = sum(scores[1:])/scores[0]
    for i in scores[1:]:
        if i > avg:
            num += 1
    print(f'{(num/scores[0])*100:.3f}%')

File: 백준/Bronze/4344/4344.js
const fs = require('fs');
const input = fs.readFileSync('input.txt').toString().trim().split('\n');

for (let i = 1; i <= parseInt(input[0]); i++) {
  let lineInput = input[i].split(' ');
  let scores = lineInput.slice(1).map(score => parseInt(score));
  let sum = scores.reduce((sum, score) => sum + score, 0);
  let avg = sum / scores.length;
  
  let overAvg = scores.filter(score => score > avg);
  let proportion = overAvg.length / scores.length;
  console.log((proportion*100).toFixed(3) + '%');
}